Tools/RemoteConsole/ly_remote_console/ly_remote_console/remote_console_commands.py

```diff
@@ -137,13 +138,13 @@ def start(self, timeout=10, timeout_port=30, retry_delay=5):
         # Check if the remote console is ready
         self.pump_thread.start()
         if not self.ready.wait(timeout):
-            raise Exception("remote_console_commands.py:start: Remote console connection never became ready. "
-                            "Waited for {} seconds.".format(timeout))
+            raise Exception(f'remote_console_commands.py:start: Remote console connection never became ready. Waited for {timeout} seconds.')
 
         self.connected = True
-        logger.info('Remote Console Started at port {}'.format(self.port))
+        logger.info(f'Remote Console Started at port {self.port}')
+        print(f'Remote Console Started at port {self.port}')
```

Comment:
Remove duplicate print

Reply:
Well the point is that the logger logs to a file, it is silent. You see nothing in the console, if there is a sucessful connection and logging started or there is an error and it continues connection attempts or there is a failure and it is stuck, no log captured. I added those duplicates intentionally, otherwise there is a high chance of silent failure.